modules/swagger-codegen/src/main/resources/META-INF/services/io.swagger.codegen.CodegenConfig

```diff
@@ -7,7 +7,9 @@ io.swagger.codegen.languages.FlaskConnexionCodegen
 io.swagger.codegen.languages.GoClientCodegen
 io.swagger.codegen.languages.JavaClientCodegen
 io.swagger.codegen.languages.JavascriptClientCodegen
-io.swagger.codegen.languages.JaxRSServerCodegen
+io.swagger.codegen.languages.JavaClientCodegen
+io.swagger.codegen.languages.JavaJerseyCodegen
+io.swagger.codegen.languages.JavaCXFCodegen
```

Comment:
OK. I'll make the change.

(technically you can also submit a PR against my repo)